Formula/cloudflared.rb

```diff
@@ -0,0 +1,27 @@
+class Cloudflared < Formula
+  desc "Cloudflare Tunnel client (formerly Argo Tunnel)"
+  homepage "https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide"
+  url "https://github.com/cloudflare/cloudflared/archive/refs/tags/2022.2.2.tar.gz"
+  sha256 "28bccb9a99cfcccade673f1c14cdbecc5fba7ab58fa6f1062cd671d749904ba5"
+  license "Apache-2.0"
+  head "https://github.com/cloudflare/cloudflared.git", branch: "master"
+
+  depends_on "gettext" => :build
+  depends_on "go" => :build
+
+  def install
+    system "make", "cloudflared"
+    system "sh", "-c",
+      'DATE="\\"$(date -u \'+%Y-%m-%d-%H%M UTC\')\\"" '\
+      'VERSION="$pkgver" '\
+      "envsubst < cloudflared_man_template > cloudflared.1"
```

Comment:
Can we use `with_env` and the `Formula` `time` method here? It'll save an `sh` call (which is more efficient), and will use a date that doesn't depend on the exact time the build runs (which is more reproducible).

Reply:
Should be like this, but I'm typing this in the web editor
```suggestion
    with_env(DATE: time.iso8601, VERSION: version) do
      system  "envsubst < cloudflared_man_template > cloudflared.1"
    end
```

Reply:
Thanks for the tip!